src/tasks/keycloak/config.ts

```diff
@@ -46,7 +47,28 @@ export const idpMapperTpl = (name: string, alias: string, role: string, claim: s
   },
 })
 
-export const clientScopeCfgTpl = (protocolMappers: object): object => ({
+export const adminUserCfgTpl = (username: string): Record<string, unknown> => ({
+  username,
+  clientRoles: ['manage-realm'],
+  realmRoles: ['otomi-admin'],
+})
+
+export const realmCfgTpl = (realm: string): Record<string, unknown> => ({
+  id: realm,
+  realm,
+  displayName: realm,
```

Comment:
Can make displayName configurable and do not point to out realm that is used in URLS